src/components/attendee/EventInfo/EventInfo.tsx

```diff
@@ -0,0 +1,50 @@
+import ShowMoreText from "react-show-more-text";
+import cn from "classnames";
+
+import { STRING_DASH_SPACE, STRING_SPACE } from "settings";
+
+import { WorldEvent } from "types/venues";
+
+import { eventEndTime, eventStartTime } from "utils/event";
+import { formatTimeLocalised } from "utils/time";
+
+import { RenderMarkdown } from "components/organisms/RenderMarkdown";
+
+import CN from "./EventInfo.module.scss";
+
+interface EventInfoProps {
+  event: WorldEvent;
+  centered?: boolean;
+}
+
+const moreLessSpan = (text: string) => (
+  <span className={CN.moreLessSpan}>{text}</span>
+);
+
+export const EventInfo: React.FC<EventInfoProps> = ({ event, centered }) => (
+  <div data-bem="EventInfo" className={cn(CN.general, centered && CN.centered)}>
+    <h3 className={CN.paragraph}>{event.name}</h3>
+
+    {event.host && <div className={CN.paragraph}>By {event.host}</div>}
+
+    <div className={CN.paragraph}>
+      <span>
+        {formatTimeLocalised(eventStartTime({ event })) + STRING_DASH_SPACE}
+      </span>
+      {STRING_SPACE}
+      <span>{STRING_SPACE + formatTimeLocalised(eventEndTime({ event }))}</span>
+    </div>
+
+    <div className={CN.paragraph}>
+      <ShowMoreText
+        lines={5}
+        anchorClass={CN.showMoreLessAnchor}
+        more={moreLessSpan("More")}
+        less={moreLessSpan("Less")}
+        expanded={false}
```

Comment:
[Question] Do we need to include a falsy property at all? Why don't we provide a default `expanded = false` inside the `ShowMoreText` to avoid unnecessary props?

Reply:
yes, good catch, thanks. done: https://github.com/sparkletown/sparkle/pull/3071/commits/53ad693cb533a0eaa1fddc004d2ab374630c2be1